entity-service/src/main/kotlin/com/egm/stellio/entity/service/EntityService.kt

```diff
@@ -268,61 +278,61 @@ class EntityService(
      */
     fun getFullEntityById(entityId: String): ExpandedEntity {
         val entity = entityRepository.getEntityCoreById(entityId)
-        val resultEntity = entity.serializeCoreProperties()
+        var resultEntity = entity.serializeCoreProperties()
 
         // TODO test with a property having more than one relationship (https://redmine.eglobalmark.com/issues/848)
         entityRepository.getEntitySpecificProperties(entityId)
-            .groupBy {
-                (it["property"] as Property).id
-            }
-            .values
-            .forEach {
-                val (propertyKey, propertyValues) = buildPropertyFragment(it, entity.contexts)
-                resultEntity[propertyKey] = propertyValues
-            }
+        .groupBy {
+            (it["property"] as Property).id

```

Comment:
Will this really work ? I'm afraid if there are multiple properties returned by getEntitySpecificProperties this will be an error, right ?

Reply:
Actually the properties will not have the same id.
I don't see how it could raise an error. 
@bobeal, any ideas ?